README.md

```diff
@@ -188,6 +184,21 @@ It is hard knowing what is going on inside of some of these ADTs or your wonderf
 #### `tryCatch : (a -> b) -> a -> Either e b`
 Typical try-catch blocks are very imperative in their usage. This `tryCatch` function provides a means of capturing that imperative nature in a simple declarative style. Pass it a function that could fail and it will return you another function wrapping the first function. When called the new function will either return the result in an `Either.Right` if everything was good, or an error wrapped in an `Either.Left` if it fails.
 
+### Logic Functions
+The functions in this section are used to represent simple to complex logical branching in a declarative manner. All of these function will take predicate functions or `Pred`s and either return a resulting `Boolean` or run some function based on the result of the predicates. All of these functions work with functions/`Pred`s instead of values to allow for building out complex, "lazy" evaluated and allow easy composition.
```

Comment:
All of these function[s] will take predicate functions (crocks says`Pred`s), [and either return a `Boolean` or run some function based on the result of the predicates === not informative, readers' deduction should work here].